### ISSUE TITLE
WalletLink and Metamask: Uniswap artefact 

### ISSUE THREAD

**Issue by MareenG** [gnosis/cowswap#317]
If I monitor my connections in the Coinbase Wallet it states that I am connected to Uniswap not Cowswap. This accounts only for the Logo and the name the URL is the correct one

![1616488030512](https://user-images.githubusercontent.com/34510341/112116682-d04fca00-8bba-11eb-9ce4-c1555bf601fb.jpg)

**Comment by MareenG** [comment]
Another Uniswap artefact in the MM pop up

When changing between networks in MM the pop up show different brandings: 
Connecting for the first time: 
<img width="340" alt="Bildschirmfoto 2021-03-24 um 12 18 42" src="https://user-images.githubusercontent.com/34510341/112302577-8985d100-8c9b-11eb-97a1-316ff2d5d572.png">
Changing to a different Network:
<img width="330" alt="Bildschirmfoto 2021-03-24 um 12 20 05" src="https://user-images.githubusercontent.com/34510341/112302584-8ab6fe00-8c9b-11eb-96f6-314563637c7c.png">
<img width="322" alt="Bildschirmfoto 2021-03-24 um 12 20 20" src="https://user-images.githubusercontent.com/34510341/112302588-8b4f9480-8c9b-11eb-9e3b-b678e063e796.png">

Note: 
In the last days I didn't have to approve the MM connection when changing the Network. I guess some changes are done currently?

**Comment by anxolin** [comment]
@MareenG are you sure about the Metamask thing? It clearly shows different URLs... 
I can't reproduce. Would you give me the steps?

**Comment by MareenG** [comment]
@anxolin yes, I am quite sure about  the MM thing. 
It puzzled me quite a lot on Wednesday. Especially, as my wallet got connected to Uniswap while interacting with Cowswap. 
However, I only experienced this on Wednesday. 
Before I never came across this issue also yesterday I didn't experienced it anymore.

**Comment by W3stside** [comment]
As an update: I can't reproduce this :/ would consider closing if no one else can reproduce

**Comment by MareenG** [comment]
In WalletLink the Uniswap artefacts are still present. The other issues is solved and seemed to be due to some changes while testing

**Comment by anxolin** [comment]
Ok, thanks. @MareenG for now, we will focus on finishing everything for our release, and we will remove all the wallets except metamask. Then we we will enable then one by one.

I initiated some PR for the Wallet Link issue, but for now we won't put too much effort in that: https://github.com/gnosis/gp-swap-ui/pull/429

We also created this PR to disable the wallets: https://github.com/gnosis/gp-swap-ui/pull/430

Additionally we created one issue per wallet so we can bring them back later.
